sys/include/fs/littlefs2_fs.h

```diff
@@ -74,6 +74,13 @@ extern "C" {
  * of wear leveling. -1 disables wear-leveling. */
 #define CONFIG_LITTLEFS2_BLOCK_CYCLES       (512)
 #endif
+
+#ifndef CONFIG_LITTLEFS2_MIN_BLOCK_SIZE_EXP
+/**
+ * The exponent of the minimum acceptable block size in bytes (2^n).
+ * The desired block size is not guaranteed to be applicable but will be respected. */
+#define CONFIG_LITTLEFS2_MIN_BLOCK_SIZE_EXP (-1)
```

Comment:
So shouldn't this be 7 if the minimum size is 104 bytes?

Reply:
Having the default block size equal to the sector size is what we want actually, because else if for some people the block size on an existing FS changes, because we choose 128 as a default value, they will complain justifiably that they are unable to read their data anymore.
I agree we could warn people if they choose something between [0, ..., 6], but LFS does not stop you from doing this, it just fails at some point. So should we issue a compilation warning or do you suggest something else?